Sketch (3).java

```diff
@@ -0,0 +1,77 @@
+import processing.core.PApplet;
+
+public class Sketch extends PApplet {
+	
+	
+  /**
+   * Called once at the beginning of execution, put your size all in this method
+   */
+  public void settings() {
+	// put your size call here
+    size(200, 200);
+  }
+
+  /** 
+   * Called once at the beginning of execution.  Add initial set up
+   * values here i.e background, stroke, fill etc.
+   */
+  public void setup() {
+    background(32, 131, 266);
+  }
+
+  /**
+   * Called repeatedly, anything drawn to the screen goes here
+   */
+  public void draw() {
+  //Ground
+    fill(0, 100, 0);
+    rect(width / 400, (float) (height / 1.333333), (float)      (width / 0.898876404494), (float) (height / 1.538461));
+
+  //Sun
+    fill(255, 255, 0);
+    ellipse(width / 400, height / 400, (float)(width /     
+    2.22222222), (float)(height / 2.22222222));
```

Comment:
You have a lot of number reused throughout your code, i.e `2.222222222`.  This makes for the perfect opportunity to define a variable so you don't have to keep restating the numerical values.